docs/08-developer-guide/02-run/02-docker-compose.md

```diff
@@ -10,3 +10,143 @@ sidebar_label: Docker Compose
 - Define the Ozone runner image at `ozone-docker-runner` and its purpose to wrap Ozone binaries.
 - How to define which image gets used in the cluster (runner or other pre-built image).
 - Changing configurations in Docker Compose (`docker-config` file, transformation.py, etc).
+
+This guide explains how to run Apache Ozone using Docker Compose, either with locally built sources or pre-built images.
+
+## Prerequisites
+
+- Docker Engine 20.10.0 or higher
+- Docker Compose V2
+- Built Ozone distribution (if running from local build)
+
+## Running Ozone
+
+You can run Ozone either using your locally built version or using official pre-built Docker images.
+
+### Option 1: Running from Local Build
+
+If you've built Ozone from source, follow these steps:
+
+1. Navigate to the compose directory in your build output:
+
+    ```bash
+    cd hadoop-ozone/dist/target/ozone-*-SNAPSHOT/compose/ozone
+    ```
+
+2. Start the cluster:
+
+    ```bash
+    docker-compose up -d
+    ```
+
+The local build uses the `ozone-docker-runner` image, which is automatically created during the build process to wrap your compiled Ozone binaries.
+
+### Option 2: Running from Pre-built Images
+
+If you haven't built Ozone locally, you can quickly start a cluster using official pre-built images from Docker Hub:
+
+1. Create a directory for your Ozone deployment:
+
+    ```bash
+    mkdir ozone && cd ozone
+    ```
+
+2. Download the example compose file:
+
+    ```bash
+    curl \
+   https://raw.githubusercontent.com/apache/ozone/master/hadoop-ozone/dist/src/main/compose/ozone/docker-compose.yaml \
+   -o docker-compose.yaml
+    ```
+
+3. Start the cluster:
+
+    ```bash
+    docker-compose up -d
+    ```
+
+This will pull the official Apache Ozone images from Docker Hub.
+
+## Architecture Diagram
```

Comment:
To avoid duplicate content I think we should leave this information for the page dedicated to Ozone architecture. Default ports will be covered in `docs/administrator-guide/configuration/basic/network/`. This may mean we don't need to add mermaid in this PR.

Reply:
As a new user, it is not immediately apparent how the Ozone services will be divvied across containers. It could be the case that all Ozone services run within one container. There are several possible configurations.

I added the mermaid diagram to visualize how many containers will be available and how each Ozone service maps to those containers. 

It just so happens that each Ozone service is a separate container and the container diagram looks similar to what could easily have been an Ozone architecture diagram. Perhaps the title and wording add to the confusion.

If it is still a cause of confusion, I can remove the diagram. Not a problem.